lib/task.js

```diff
@@ -599,10 +598,11 @@ export default class Task {
       return await this.abortRun(this.taskState);
     }
     // Validate the schema!
-    let payloadErrors =
-      this.runtime.schema.validate(this.task.payload, PAYLOAD_SCHEMA);
+    let payloadErrors = await validator().then((validator) => {
```

Comment:
See change above about global validator instance... then this becomes:
let payloadErrors = this.runtime.validator.check(this.task.payload, PAYLOAD_SCHEMA);